.github/workflows/sendJsonFile.yml

```diff
@@ -78,22 +72,40 @@ jobs:
       with:
         issue-number: ${{ github.event.issue.number }}
         body-file: ./validationErrors.md
-    - name: Create Pull Request
-      id: cpr
-      uses: peter-evans/create-pull-request@v4
+    - name: Checkout transformation repo for nvdaAPIVersions.json
+      uses: actions/checkout@v4
       with:
-        path: datastore
-        title: ${{ steps.get-data.outputs.issueTitle }}
-        branch: ${{ github.event.sender.login }}${{ steps.get-data.outputs.issueNumber }}
-        commit-message: ${{ steps.get-data.outputs.issueTitle }}
-        body: "Closes #${{ steps.get-data.outputs.issueNumber }}"
-        author: github-actions <github-actions@github.com>
+        repository: nvaccess/addon-datastore-transform
+        path: transform
+    - name: Validate metadata
+      id: getAddonFileName
+      run: |
+        cd datastore
+        git add .
+        $addonFileName = git diff --cached --name-only
```

Comment:
this seems fragile - what happens on PRs with multiple file names?
Shouldn't this be tied to the validation checks that only a single file is added a nd no files are modified?